src/conways/point.rs

```diff
@@ -29,3 +29,28 @@ impl Point {
         neighbours
     }
 }
+
+impl From<&str> for Point {
+    fn from(s: &str) -> Self {
+        let mut positions = s.split(',');
+
+        let (Some(x_position), Some(y_position), None) =
+            (positions.next(), positions.next(), positions.next())
+        else {
+            return Self::default();
+        };
+
+        let Ok(x_position) = x_position.parse() else {
+            return Self::default();
+        };
+
+        let Ok(y_position) = y_position.parse() else {
+            return Self::default();
+        };
+
+        Self {
+            x_position,
+            y_position,
+        }
+    }
+}
```

Comment:
I believe it would be better to implement [`TryFrom`](https://doc.rust-lang.org/std/convert/trait.TryFrom.html) instead, because the conversion `str` to `Point` can fail if the string is not in the expected format.

Reply:
That's true, i'm going to add some tests also

Reply:
Nice!